src/Test/Utilities/Shared/Assert/ConditionalFactAttribute.cs

```diff
@@ -44,8 +44,7 @@ public class HasShiftJisDefaultEncoding : ExecutionCondition
 
     public class IsEnglishLocal : ExecutionCondition
     {
-        public override bool ShouldSkip =>
-                System.Globalization.CultureInfo.CurrentCulture != new System.Globalization.CultureInfo("en-US");
+        public override bool ShouldSkip => System.Globalization.CultureInfo.CurrentUICulture.ToString() != "en-US";
```

Comment:
```!System.Globalization.CultureInfo.CurrentUICulture.Name.StartsWith("en", StringComparison.OrdinalIgnoreCase)```?

To be consistent with code in src\Test\Utilities\Shared\FX\EnsureEnglishUICulture.cs